### PR TITLE
Make ci work

### PR DESCRIPTION
**Pull request by pngwn**
## Description

Please include a concise summary, in clear English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #(issue)

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: .github/actions/install-all-deps/action.yml [hunk 1]
```diff
@@ -37,24 +65,34 @@ runs:
...
+          gradio-lib-${{inputs.os}}-latest-pip-
+        key: gradio-lib-${{inputs.os}}-latest-pip-${{ hashFiles('client/python/requirements.txt') }}-${{ hashFiles('requirements.txt') }}-${{ hashFiles('test/requirements.txt') }}-${{ hashFiles('client/python/test/requirements.txt') }}
     - name: Install Gradio and Client Libraries Locally (Linux)
       shell: bash
       run: |
-        . venv/bin/activate
-        python -m pip install -e client/python
-        python -m pip install -e .
+        . ${{ env.VENV_ACTIVATE }}
+        python -m pip install client/python
```
Comment (freddyaboulton) [inline .github/actions/install-all-deps/action.yml:76]: I think we need to keep the editable install here for the functional tests?d

Comment (pngwn) [reply .github/actions/install-all-deps/action.yml]: thanks!

File: .github/workflows/backend.yml [hunk 2]
```diff
@@ -155,90 +47,26 @@ jobs:
...
-      run: |
-        . venv/bin/activate
-        bash scripts/install_test_requirements.sh
-    - name: Install ffmpeg
-      uses: FedericoCarboni/setup-ffmpeg@v2
-    - name: Lint (Linux)
+        python_version: ${{ matrix.python-version }}
+        os: ${{ matrix.os }}
+        test: true
+    - name: log venv activate
```
Comment (freddyaboulton) [inline .github/workflows/backend.yml:57]: does this actually do anything?

File: .config/playwright-setup.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ const TEST_APP_PATH = join(__dirname, "./test.py");
 const TEST_FILES_PATH = join(__dirname, "..", "js", "app", "test");
 const ROOT = join(__dirname, "..");
 
+console.log(ROOT);
```
Comment (pngwn) [inline .config/playwright-setup.js:14]: Remove

File: .config/playwright-setup.js [hunk 4]
```diff
@@ -107,6 +109,7 @@ function kill_process(process) {
 function make_app(demos, port) {
 	return `import gradio as gr
 import uvicorn
+import sys
```
Comment (pngwn) [inline .config/playwright-setup.js:112]: Remove

File: .github/workflows/deploy-chromatic.yml
Comment (pngwn) [inline .github/workflows/deploy-chromatic.yml:1]: Remove file

File: .github/workflows/deploy-website.yml [hunk 5]
```diff
@@ -1,108 +1,125 @@
...
+    needs: changes
     uses: "./.github/workflows/comment-queue.yml"
     secrets:
-      gh_token: ${{ secrets.gh_token }}
+      gh_token: ${{ secrets.COMMENT_TOKEN }}
     with:
-      pr_number: ${{ inputs.pr_number }}
+      pr_number: ${{ needs.changes.outputs.pr_number }}
       message: website~pending~null
   deploy:
```
Comment (pngwn) [inline .github/workflows/deploy-website.yml:45]: Rename step id

File: .github/workflows/deploy-website_old.yml
Comment (pngwn) [inline .github/workflows/deploy-website_old.yml:1]: Remove file

**Pull request merged**